src/backends/onnx/loader.cc

```diff
@@ -48,8 +49,13 @@ OnnxLoader::Init()
     OrtEnv* env;
     // If needed, provide custom logger with
     // ort_api->CreateEnvWithCustomLogger()
-    OrtStatus* status =
-        ort_api->CreateEnv(ORT_LOGGING_LEVEL_WARNING, "log", &env);
+    OrtStatus* status;
+    if (LOG_VERBOSE_IS_ON(1)) {
```

Comment:
I thought you were only going to turn on ONNX verbose at level 2 or higher?

Reply:
Done